luigi/contrib/spark.py

```diff
@@ -237,8 +237,6 @@ class PySparkTask(SparkSubmitTask):
 
     # Path to the pyspark program passed to spark-submit
     app = os.path.join(os.path.dirname(__file__), 'pyspark_runner.py')
-    # Python only supports the client deploy mode, force it
-    deploy_mode = "client"
```

Comment:
Was this an intentional deletion? Why not just allow overwrite of `deploy_mode`?